internal/pkg/cli/dialog/create.go

```diff
@@ -17,7 +17,7 @@ import (
 
 type createDeps interface {
 	Options() *options.Options
-	Components() (model.ComponentsMap, error)
+	Components() (*model.ComponentsMap, error)
```

Comment:
Aby som mohol v pripade chyby vratit jednoducho `nil`,.... je to immutable, read only, takze je to jedno.